build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -15,19 +15,19 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" Version="$(CLI_MSBuild_Version)" />
+    <PackageReference Include="Microsoft.Build" Version="15.4.8" />
```

Comment:
Why are we pinning these versions? In theory, we don't really need to keep them moving forward, as this is our build project, but I am curious anyways.

Reply:
@livarcocc 
I was caught in a small paradox. To wit: we need the dotnet-cli-build dll built before using the 'download' task to download the props file & we need the props file before we build the dotnet-cli-build dll. 

1) since the move to repo toolset is likely to replace the download task
2) since I do not want anyone to think that these versions would be overridden
3) since this is how we build the product [tooling] and not the product itself
therefore, it seems right to pin them explicitly to a particular version.